docs/Diagnostics/HealthChecker/AMSIIntegration.md

```diff
@@ -0,0 +1,30 @@
+---
+title: AMSI Check
```

Comment:
Going to need to change this page now with the new github pages formatting

Reply:
Done.